### PR TITLE
Frontend Plugins DOCS

### PR DESCRIPTION
**Pull request by okbel**
## What does this PR do?
Adds DOCS for Building Frontend Plugins

### CODE REVIEW

**Review by kgardnr — APPROVED** [review]
LGTM!!! Well done @okbel!

File: docs/frontend/PLUGINS.md [hunk 1]
```diff
@@ -0,0 +1,208 @@
...
+```
+
+### Import Actions from Talk
+We can easily trigger `Talk` actions in our plugin Components.
+
+```js
+import React, {Component} from 'react';
+import {connect} from 'react-redux';
+import {bindActionCreators} from 'redux';
+import {addTag, removeTag} from 'coral-plugin-commentbox/actions';
```
Comment (alinhle) [inline docs/frontend/PLUGINS.md]: At one point we'll need to move this to another path right? Then plugins depending on this will break.

Comment (alinhle) [reply docs/frontend/PLUGINS.md]: Or will this be part of our public API as it is?

Comment (alinhle) [reply docs/frontend/PLUGINS.md]: If not, we should add a comment somewhere pointing out that things are still under development and are subjected to change.

Comment (jde) [reply docs/frontend/PLUGINS.md]: Sorry for the delayed response to this. Agreed that we cannot expose core functionality to plugin developers without a layer that will allow us to make updates without breaking plugins. I'm working on an update that will clarify.

**Pull request merged**